core/src/contracts/stablex_contract.rs

```diff
@@ -324,6 +323,33 @@ impl StableXContract for StableXContractImpl {
             .stream()
             .boxed()
     }
+
+    fn noop_transaction<'a>(&'a self, gas_price: U256) -> BoxFuture<'a, Result<TransactionResult>> {
+        use futures::compat::Future01CompatExt as _;
+        async move {
+            let web3 = self.instance.raw_instance().web3();
+            let account = self.account().ok_or_else(|| anyhow!("no account"))?;
+            let address = account.address();
+            let nonce = web3
+                .eth()
+                .transaction_count(address, None)
+                .compat()
+                .await
+                .context("failed to get nonce")?;
+            let transaction_result =
+                ethcontract::contract::MethodBuilder::fallback(web3, address, Default::default())
+                    .from(account)
+                    .gas(U256::from(21000))
```

Comment:
Is this the right gas amount?

Reply:
AFAIK that is the base transaction gas amount.